samples/DurableFunctions/README.md

```diff
@@ -0,0 +1,326 @@
+# Create your first C# Durable Functions on Azure Container Apps
+
+[Durable Functions](https://learn.microsoft.com/azure/azure-functions/durable/durable-functions-overview) is an offering of Azure Functions that lets you write stateful functions in a serverless environment. The extension manages state, checkpoints, and restarts for you.
+
+This quickstart will show you how to create a Durable Function that runs on Azure Container Apps.
```

Comment:
nit - now that the project has been pushed, should we update the README to say that these instructions yield the sample in `LocalFunctionProj`